src/routes/posts/+page.svelte

```diff
@@ -3,10 +3,73 @@
 	import { parseContent } from '$lib/parse';
 
 	export let data;
+
+	$: category = data.category ?? null;
+	$: start = data.start ?? null;
+	$: end = data.end ?? null;
 </script>
 
 <section style="max-width: 1200px; margin: 0 auto; ">
 	<h3>Recent Articles without Alt Text</h3>
+	<div
+		style="display: flex; flex-wrap: wrap; justify-content: center; align-items: center; gap: 3rem;"
+	>
+		<div>
+			<label for="category">Select category:</label>
+			<select id="category" bind:value={category} on:selectionchange>
+				<option value={null}>All</option>
+				<option value={46}>News</option>
+				<option value={44}>Sports</option>
+				<option value={31}>Opinion</option>
+				<option value={5}>Arts</option>
+				<option value={55}>Statement</option>
+				<option value={32}>MiC</option>
+				<option value={124}>Photos</option>
+				<option value={24}>Podcasts</option>
+				<option value={10327}>Videos</option>
+			</select>
+		</div>
+		<div>
+			<label for="start"
+				>From
+				<input id="start" type="date" bind:value={start} />
+			</label>
+			<label for="end">
+				to
+				<input id="end" type="date" bind:value={end} />
+			</label>
+		</div>
+		<a
+			id="apply"
+			href={`posts?page=${data.page}
+				${category ? `&category=${category}` : ''}
+				${start ? `&start=${start}` : ''}
+				${end ? `&end=${end}` : ''}`}
+		>
+			Apply Filter
+		</a>
+	</div>
+
+	<nav
+		style="display: flex; justify-content: center; gap: 15px; padding: 1rem; font-size: 1.25rem;"
+	>
+		{#if data.page > 0}
+			<a
+				href={`/posts/?page=${data.page - 1}
+				${category ? `&category=${data.category}` : ''}
+				${start ? `&start=${data.start}` : ''}
+				${end ? `&end=${data.end}` : ''}`}>← Back</a
+			>
+		{/if}
+		{#if data.articles.length > 0}
+			<a
+				href={`/posts/?page=${data.page + 1}
+				${category ? `&category=${data.category}` : ''}
+				${start ? `&start=${data.start}` : ''}
+				${end ? `&end=${data.end}` : ''}`}>Next →</a
+			>
+		{/if}
+	</nav>
 	<ul>
 		{#each data.articles as article}
```

Comment:
Might be helpful in the case that there are no articles to show some sort of message using the each-else syntax: <https://svelte.dev/repl/92c631981a4c46fc8e96f22a9ca09f47?version=3.15.0>